ci/pipeline

```diff
@@ -221,3 +236,45 @@ def jenkins(): Unit = {
   if(utils.isPullRequest) pr()
   else master()
 }
+
+/**
+ * Executes the Community Release which includes:
+ *  1. tarball with version details
+ *  2. release of docker with that version
+ *  3. git tag of version (pushed to server)
+ *
+ * @return Version and artifact description of Marathon build.
+ */
+@main
+def release(requestVersion: String, gitSha: String, latest: String = "False"): Unit = {
+
+  val releaseVersion = cleanVersion(requestVersion)
+  val tagVersion = s"v$releaseVersion"
+  println(s"Releasing version: $releaseVersion ")
+  %('git, "checkout", gitSha)
+  %('git, "tag", "-a", tagVersion, "-m", tagVersion)
+  val version = build()
+
+  uploadTarballPackagesToS3(version, s"releases/$releaseVersion")
+
+  %('docker, "push", s"mesosphere/marathon:$tagVersion")
```

Comment:
What are the differences between `tagVersion`, `version` and `releaseVersion`? Shouldn't they all be the same?

Reply:
`requestVersion` is the request attempt and will be cleaned to be the `releaseVersion`
`releaseVersion`  is the version with out the `v` prefix
`tagVersion` is the version with the `v` prefix
`version` could go away... it is the version determined by the underlying build which should be the same as tagVersion.   It is there being consistent with the other builds (master and pr) which require it because the version isn't passed in but is calculated.

Reply:
We should document this 🙂.